jabgui/src/main/java/org/jabref/gui/theme/ThemeManager.java

```diff
@@ -49,13 +55,16 @@ public class ThemeManager {
     );
 
     private static final Logger LOGGER = LoggerFactory.getLogger(ThemeManager.class);
+    private static final boolean SUPPORTS_DARK_MODE = OS.WINDOWS || OS.OS_X;
```

Comment:
Please comment why it is not supported on Linux. Add a link. I think, KDE supports dark mode, doesn't it?

Reply:
This reminded me that there is a thing they are working on ttps://bugs.openjdk.org/browse/JDK-8313424

Reply:
> Would it be better for me to just have called the stub methods anyway

Yes, I think so. Maybe, there are some Window managers we never thought of supported? Icewm, dwm, ...